lib/membrane_g711_ffmpeg/decoder.ex

```diff
@@ -57,8 +61,18 @@ defmodule Membrane.G711.FFmpeg.Decoder do
   def handle_stream_format(:input, stream_format, _ctx, state) do
     encoding =
       case stream_format do
-        %G711{encoding: encoding} -> encoding
-        %RemoteStream{} -> state.encoding
+        %G711{encoding: encoding} ->
+          unless encoding in [nil, state.encoding] do
```

Comment:
I don't think we want that `nil` here, as later we use that `encoding` as an argument for `Native.create/1`